docs/articles/remoting/transports.md

```diff
@@ -111,8 +111,8 @@ akka{
 
 Both TCP and UDP are enabled in this scenario. But how do I know which transport is being used when I send a message to a `RemoteActorRef`? That's indicated by the protocol scheme used in the `Address` of the remote actor:
 
-    akka.tcp://MySystem@localhost:8081/user/actor #helios.tcp
-    akka.udp://MySystem@localhost:8082/user/actor #helios.udp
+    akka.tcp://MySystem@localhost:8081/user/actor #dot-netty.tcp
+    akka.udp://MySystem@localhost:8082/user/actor #dot-netty.udp
```

Comment:
Yep. Im going to change that to a fictional protocol, so we still illustrate the point we are making there. But not give false expectations